docs/changelog.rst

```diff
@@ -7,15 +7,30 @@ Changelog
 Next Version
 ------------
 
-- fixed a bug where a dataprovider could upload her clks multiple time in a project using the same upload token (#463)
+
+Version 1.13.0-beta
+-------------------
+
+- fixed a bug where a dataprovider could upload their clks multiple time in a project using the same upload token (#463)
```

Comment:
mix of singular and plural. Yes, we celebrate diversity, but that might take it just a tad too far.